README.rst

```diff
@@ -3,16 +3,16 @@ pypuppetdb
 ##########
 
 .. image:: https://api.travis-ci.org/puppet-community/pypuppetdb.png
-   :target: https://travis-ci.org/puppet-communit/pypuppetdb
+   :target: https://travis-ci.org/puppet-community/pypuppetdb
 
 .. image:: https://coveralls.io/repos/puppet-community/pypuppetdb/badge.png
-   :target: https://coeralls.io/r/puppet-community/pypuppetdb
+   :target: https://coveralls.io/repos/puppet-community/pypuppetdb
+
 
 pypuppetdtb is a library to work with PuppetDB's REST API. It is implemented
 using the `requests`_ library.
 
-**pypuppetdb DOES NOT work with PuppetDB 3**
-
+**pypuppetdb >= 0.2.0 DOES work with PuppetDB 3**
```

Comment:
Oh, ignore me. I read "DOES NOT" work with PuppetDB 3, but it clearly does not contain the "NOT" in the sentence. Carry on. :100: